code/game/machinery/tcomms/_base.dm

```diff
@@ -106,23 +106,37 @@ GLOBAL_LIST_EMPTY(tcomms_machines)
 
 
 /**
-  * Start of Ion Anomalie Event
+  * Start of Ion Anomaly Event
   *
-  * Proc to easily start an Ion Anomalie's effects, and update the icon
+  * Proc to easily start an Ion Anomaly's effects, and update the icon
   */
 /obj/machinery/tcomms/proc/start_ion()
 	ion = TRUE
 	update_icon()
 
 /**
-  * End of Ion Anomalie Event
+  * End of Ion Anomaly Event
   *
-  * Proc to easily stop an Ion Anomalie's effects, and update the icon
+  * Proc to easily stop an Ion Anomaly's effects, and update the icon
```

Comment:
These typos were hurting me so I fixed them

Reply:
Who is Anomalie and why is she making these ion anomalies?!